src/gui/pages_menu/KM_GUIMenuLobby.pas

```diff
@@ -53,7 +53,8 @@   TKMMenuLobby = class
 
     procedure PlayersSetupChange(Sender: TObject);
     procedure MapColumnClick(aValue: Integer);
-    procedure MapTypeChange(Sender: TObject);
+    procedure MapTypeChange(Sender: TObject); overload;
+    procedure MapTypeChange; overload;
```

Comment:
Why is this overloaded? What is the difference between the two versions? Presumably the Sender one is used when the radio is clicked, but then what's the other one for? It's not clear at all. If it serves a different purpose maybe it should have a different name.